docs/DeveloperGuide.md

```diff
@@ -37,8 +37,8 @@ The rest of the App consists of four components.
 
 * [**`UI`**](#ui-component): The UI of the App.
 * [**`Logic`**](#logic-component): The command executor.
-* [**`Model`**](#model-component): Holds the data of the App in memory.
-* [**`Storage`**](#storage-component): Reads data from, and writes data to, the hard disk.
+* [**`Model`**](#model-component): Holds the data of the App in memory. Comprises of an `InventoryModel` and a `DeliveryModel` which handles `Item` and `Delivery` respectively.
```

Comment:
"Comprises" is misleading. You could say "is implemented by InventoryModelManager and DeliveryModelManager", but frankly, it's not necessary to add that line here. The first sentence sounds enough since this is just an overview

Reply:
fixed in latest commit https://github.com/AY2021S1-CS2103T-T12-1/tp/pull/169/commits/c276c055ccc67713af31fe5506ca399aa0cee012